### DISCUSSION TITLE
TypeError: fields.flat is not a function

### DISCUSSION THREAD

**Discussion by EISCoding** [eritislami/evobot#681] [Q&A]
Can someone help me? I cant use /play or other commands.

https://prnt.sc/10pyvgo

**Comment by Takiyo0** [comment]
You need to update your node.js version because discord.js requires 12.0.0 or newer.

here's the link where i found the answer:
https://stackoverflow.com/questions/60934727/discord-js-messageembed-fields-flat-is-not-a-function
and here
https://github.com/discordjs/discord.js/issues/3913#issuecomment-596196573
